packages/modelserver-client/src/model-server-client-v2.spec.ts

```diff
@@ -36,19 +42,19 @@ describe('tests for ModelServerClientV2', () => {
 
     it('initialize - correct baseUrl config of axios instance', () => {
         const axios = client['restClient'];
-        expect(axios.defaults.baseURL).to.be.equal(baseUrl);
+        expect(axios.defaults.baseURL).to.be.equal(baseUrl.toString());
```

Comment:
as above maybe use the internal equals method?

Reply:
Thanks, will update those comparisons as well in upcoming commit.